proto/bothan/v1/bothan.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+package bothan.v1;
+
+import "google/api/annotations.proto";
+
+option go_package = "github.com/bothan/bothan-api/client/proto";
+
+// SignalService defines the gRPC service responsible for updating the registry
+// and managing active signal IDs.
+service BothanService {
+  // Gets the information regarding the signal service
+  rpc GetInfo(GetInfoRequest) returns (GetInfoResponse) {
+    option (google.api.http).get = "/info";
+  }
+
+  // Updates the registry with the given IPFS hash and version.
+  // The registry stores metadata and configuration data that can be referenced
+  // by other parts of the system.
+  rpc UpdateRegistry(UpdateRegistryRequest) returns (UpdateRegistryResponse) {
```

Comment:
also please keep the comment in the format of `UpdateRegistry is ...` or `UpdateRegistry does ...`

Reply:
formatted to  `Struct defines ...`